backend/prisma/seed.ts

```diff
@@ -59,87 +59,87 @@ async function main() {
s0JlzKYJRcuke6B3gwI8q9HxpKSs5s12jzLH4Vw83RlPKToByiJ8K0XZfsvauI9/EyUB7qpm7076AggbfOiuNwiO+XMHWTmJECW8pjfeokxZwzhIOTcc2KmNVOxGh+e3KyzeiX4bdmmGFwgti2iIgMlYRVfeMwEZtPvE61k7eGRL+YFYzkMDMy0jSTvzE9KkXHvcf2jqQnug6ByqljoSJEnSQaK2CzuXRCgkQFEFhoZJ6xz8anmyxjT9lIY7sfc4zlISMwO7HuEgkjQjT9eFDMEWQse9mYnWZnXu/Db9RR8WVcHOIImQAJ/tNS4ThSsQQI0riy+RLIuMSsccY7fY/hdtmYZjJ/XStlgMPHe+H5mh3C+FwQeXP+wo+BXX/TvGlG5TW7ObyMieoioFiME7PKEJm1cwpMCQImefKOp50erkV7BzQm4O0Q37oRGdzoikk+AFee2Jv3VRtc7teueWY5FPrMf/XWh7a8QCW/ZyBPefwUa6+onyHlWf4Jwu9piVbKzHVG2yDRV84+ZNGtC/ZqzYTaPiTWP7UYX2csCTmR4GojaDqfAjlWyWG23HpHnWI7b4mbjL9xFWOWveP/AFUeEW7roHKXVm47K24wtjqbak+bCT9aM0O4MhFi0OiIPgoq+DQYUdNcFcnWnUDHGppNcYmm5TRMdgUqWQ0qNmH5xXQ81Ex6U0gnfatQLJ9KiuAdJqF/CajZmoqILK+PTMpA0P65dDt60C7N8WKu4dXRTddQrToEZVDoTrkIZD0GbpR1zPL6RWP4+oS8jMgZES7NwkgWwwDQVAhlOXQwCMm9M0BHo4Ndis32c4uCBZc6x3DMyOhPM7weY8pOkqbVDCio7tkMPHrUtKlaTVMK0Z7H8M1zAQdvA0IxaNlCwJ6xpHh+udbgrOhqpf4erDSvMz+DytxOzF5VUpHlL2WzF1JUidQTP8sVTXBPAJVgCSNdRPI+delX+AqNkqrd4cSAsQBXnvFnx6o6/wAuOTtGYw2HCKVXfnIMGdyapXcrDvJ7ugZRsD89zWuHBhzpNwZWEcudT45r2hoyh7Mtw/CWywEsYgiTpPiK0GGuAuFH2Qd+c7b71ZscFRW7p8wI+dGsNwcT7o8zVYYcmTSEnlhF2CsNgSxzMInkNqN4HAc4gfWr1jBqu+p+VWAa9Lx/DUFcuzjy5+WkcCxpSipN6YVrvTOUbVbH4tbSM7cthtJ5D1NVOP8AHLeFTO8kkwiKJd2+6g/PYV5b2j7TXriE3CFZhCohlbanTf7TnbNz2AgEk2FRbLOABxeIZmOZM+Zz995+SCAFHRR6+iYe1AArL9iOHZLa6a1r3MCniqW+wSdvXRSu4Ym4rq7LAylR7rDlmHrXmfHsR7R7jCNbjfASAfGNK9IxmKyI7z7iM39Kk15VhhmdF3LMo5RJZRrPnTehT2/DJCKByUD4Cp4qNDpXWakYx1W1pzGN6rPilXbX61AC7nQQPnWoxb9uv60qN8ci+fgP7004IdTQziaLbGd3yj4k+AFFJMBbPGPw/OlWNucc1MW3I5HQT86VNwNZ6BTHk0ncCoGuMfdEfE0EgD2teFQNhakzN11qULTXQAfcQjeT0igXaHDO9p8mXPkICvBRpgw4YQV7vpJrVsapYpJB6eQ+FG7Aeem4FJtqCjWwgjWIKAjIW1YaMNz7h1rRcI7XlQFuhnX7y6uvjG7jy738XIHxnCvbe7cUs6MbZFsbpGjOnUZUzZQNSp01obiVgz+p5xRpSVMPWz17h+Pt30Fy06uh+0pnUbg9COh1q3XjXCeKth73tUaC0C4p9y4PxgbOOTjXrIr1rhvEbd9A6MCOY5qejDkai4tdj2vRbFPArgFdpGY7THtqdwDTqdQavsNlZ8GhEFfrXBhE2yip2aKjFydoik/DBu6QVKVabHJbVfdUDyAFSU1TTpplFR0kBts4RXAKdFcZgBTAHbVT4jjktIzuYUepJOyqObE6AUH472rs2EZlZXYaABu7P4mEgeQknpXneJ7SXcS3fAZp7kAhVB0MIdPVpMTtMVkthS1b6Gcc4q12+9ww1wjKqgytpOSAjmd2I1Y6Du0FbBl7qKdZYEzuTv8AlRtcIEBYgZiSeWnWp+ynDzexOc+7b1Pmdh9aqoV2By+D0ThGFyIB4V3EvOgq1fbKtUkbWaP2KCO1JyYO71YKn9TAH5TWD4EmbE2EHN0JHgpzT4xpp68q2Pbm6f2ZV+9cEn+FWaPkKzvYmyWxiECcod+g0GTU+Bbp9aPox6znC7mqeIDtqAfCf7VcW0N21Py9BTmcDz6UgSnhsKdzvV8LA0qhxDi1u0DmMtyUbnp5DxNZbiHFrt45fdT7i8/4jz+lZJyNdBninaFElLfffr9keo38hWTxjXLrZnYsfkPADlRPCcHYmToPnV5uHgEwNBVUkhbM9/l3jXK0f7IOlKmMGjT1pUqmzDlQ+FP9nSpUoRjWxUF9KVKmQGZ/j3DluWmRpiJJVirAjvAqeRBAPpWNe2xa4CVIDwhEzJ3VpA1B5jSMupg0qVMZFK5Y8Kfg8VctHMjssGND+uvlSpUzAjVcP7eXFhbiK5nckqY81Ugn0FGH/wAQLCDNet3bazGaEdf9L5v9NKlUMqSWikNvZpuH4+3fQXLRzI2xgj5MAatxSpVL0H5OFZpqWgNqVKiAdXGcDWuUqyMZTi3bNFzph19o66EtKIpOuumZj4AR4isViOK3cQRndnIJkyUQGdlQHuxt9qeZNKlTcVa/gotJ/wAkDcMDNLnMR8B5DYelX7VlLY7o1686VKuiMUuiLbZVxT6E1sOwWBy2TcO7sSPIaD5g0qVafQq7DuN1MdaYuF01PwpUqkMY/wDxEvSllAO7nbXxVQCI/m3qP/DSx+8vXNu4qiYmSWLbcu6sTSpUfRjeYrGhULkwo30k1nsXx5iCtsQPvH3j/alSpopGYMt4VrhnbxJkmi+DwQQCdTXaVMxQiFyj8UadKr28QeYmlSoIJMGP3RSpUqBj/9k=",
```

Comment:
why would we encode an image to base 64, base64 makes the file bigger, the bigger, the slower to transfer it to frontend for example.

change it to raw image url like this => https://images.pexels.com/photos/1624487/pexels-photo-1624487.jpeg